storageconnector/registryconnector.go

```diff
@@ -19,7 +21,7 @@ type registryStorage struct {
 	Token string     // OAUTH2 Token
 
 	Features 	*features.Client
-	Files		*files.Client
+	Metas		*feature_metas.Client
```

Comment:
not sure if should fix

"feature_metas" actually comes from goswagger applied to "feature-metas"